### PR TITLE
Add ET links

### LINKED ISSUE #939: Estonian translation not included in language list
The 4.0 license deed is available in Estonian, but the language does not appear in the language list in the dropdown or at the bottom of the page. 

The translation is here: https://creativecommons.org/licenses/by/4.0/deed.et

@georgehari are you able to tell from Transifex how the language should be listed? (capitalized or not, etc) If not, could you please follow up with the translators to ask, and then reply on this ticket? Thanks!

### PR DESCRIPTION
**Pull request by aldenstpage**
This adds ET links to the dropdown menu and footer to the licenses. Fixes #939.

### CODE REVIEW

File: docroot/legalcode/by-nc-nd_4.0_el.html [hunk 1]
```diff
@@ -706,52 +706,6 @@ <h6><a href="https://creativecommons.org/about/contact">We'd love to hear from y
     </div>
   </footer>
 </div>
-<!-- Site Footer Start - DO NOT DELETE -->
```
Comment (kgodey) [inline docroot/legalcode/by-nc-nd_4.0_el.html]: @aldenstpage this says `DO NOT DELETE`, is it okay to remove it?

Comment (kgodey) [reply docroot/legalcode/by-nc-nd_4.0_el.html]: I see other files with the same issue too.

Comment (aldenstpage) [reply docroot/legalcode/by-nc-nd_4.0_el.html]: They're temporary anchors that get replaced by the scripts. The "DO NOT DELETE" section is for the translators working on the templates. If you check out this PR and open the files in question, you'll see that they render correctly with the footer intact.

**Comment by kgodey** [comment]
thanks @aldenstpage! please merge and deploy.

**Pull request merged**